src/cloud-api-adaptor/test/e2e/README.md

```diff
@@ -48,6 +48,38 @@ To leave the cluster untouched by the execution finish you should export `TEST_T
 
 To use existing cluster which have already installed Cloud API Adaptor, you should export `TEST_INSTALL_CAA=no`.
 
+## Attestation and KBS specific
+
+We need artificats from trustee when do attestation test, to prepare trustee, do as following. 
+
+```
+pushd ${cloud-api-adaptor}/test/e2e
```

Comment:
Optional thing I've just realised - for this and the podvm image build  ~16 lines below, I've just realised that a user could think of `${cloud-api-adaptor}` as being either the repo's root directory, or <repo root>/src/cloud-api-adaptor, so maybe it's worth clarifying this by changing it to something like `push ${cloud-api-adaptor-repo-dir}/src/cloud-api-adaptor`? for clarity

Reply:
Fixed